projects/k8s-semconv.md

```diff
@@ -0,0 +1,90 @@
+# K8s Conventions Working Group
+
+## Description
+
+Kubernetes is the leading platform for Opentelemetry Collector deployment (80.6%) according
+to a recent [survey](https://opentelemetry.io/blog/2024/otel-collector-survey/#otel-components-usage).
+The OpenTelemetry community would like to stabilize k8s semantic conventions including k8s/container metrics
+in order to help the adoption of the respective OTel Collector receivers and processors.
+The goal of this SIG project is to work towards this direction.
+
+Project Board: TBA
+
+## Roadmap
+
+* [META: Define Semantic Conventions for k8s metrics](https://github.com/open-telemetry/semantic-conventions/issues/1032)
+* [Clarify the brief of container.image.id](https://github.com/open-telemetry/semantic-conventions/issues/1236)
+* [Define rules for Kubernetes name and uid resource attributes](https://github.com/open-telemetry/semantic-conventions/issues/430)
+* [Add k8s.pod.ip attribute](https://github.com/open-telemetry/semantic-conventions/issues/1160)
+* [k8s: add metric for pod status conditions](https://github.com/open-telemetry/semantic-conventions/issues/1398)
+* [Proposal: Define mapping from k8s well-known labels to semconv](https://github.com/open-telemetry/semantic-conventions/issues/236)
+* [k8s: new attributes: CSI driver and volume handle](https://github.com/open-telemetry/semantic-conventions/issues/1119)
+* TBA
```

Comment:
A few things you'll need to work w/ Entities/Semconv group on ->

(1) Should `type` on resource semconv have *all* identifying attributes or locally identifying attributes.

E.g. `k8s.namespace` today only contains `k8s.namespace.name`.  How does the interact with `k8s.pod` type?  Do I always put both when I can?

(2) Discovery of these attributes (locally and externally) (a.k.a. multi-observer problem).  I'd like if each of the resources (and attributes) had some description in the markdown for how they can be discovered BOTH inside an SDK and externally (e.g. via collector).

Reply:
We can note them down on a SemConv issue, right? I will create one.

Reply:
I would also note [this one](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24425) which was closed from inactivity but still remains problematic

Reply:
We now have the required GC member approvals to merge this and get the project started, however we're missing the resolution of this conversation. Can we consider it resolved?

Reply:
I have added https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24425 at https://github.com/open-telemetry/community/pull/2352/files#diff-5215dd2a44e15131fd183ae7adfc71c318d40c3315ba6e5c9a4aa5159b6d8227R29 to cover @jaronoff97 's comment.

I have filed https://github.com/open-telemetry/semantic-conventions/issues/1420 to cover @jsuereth 's comment.